build-tools/automation/build.groovy

```diff
@@ -1,11 +1,22 @@
 // This file is based on the Jenkins scripted pipeline (as opposed to the declarative pipeline) syntax
 // https://jenkins.io/doc/book/pipeline/syntax/#scripted-pipeline
-def XADir = "xamarin-android"
+def XADir = 'xamarin-android'
```

Comment:
Why the use of single-quotes instead of double-quotes?  Does it really matter for this expression?  (Mostly interested in terms of reducing patch size, which might not really matter...)

Reply:
simply for code consistency with a constant variable assignment i added below it. i tend to favor single quotes in these situations. trivial to revert this back if this superfluous change causes more trouble than its worth.